daemon/algod/api/client/restClient.go

```diff
@@ -462,6 +462,14 @@ func (client RestClient) RawBlock(round uint64) (response v1.RawBlock, err error
 	return
 }
 
+// RawBlockV2 gets the raw block info for the given round
+func (client RestClient) RawBlockV2(round uint64) (response []byte, err error) {
+	var blob Blob
+	err = client.getRaw(&blob, fmt.Sprintf("/v2/blocks/%d", round), rawFormat{Format: "msgpack"})
+	response = blob
+	return
+}
```

Comment:
Could you please use the `versionAffinity` here to select between v1 and v2 ?

Reply:
done